cloudformation/media-atom-maker.json

```diff
@@ -78,9 +86,20 @@
                 "DesiredCapacity": 1,
                 "InstanceType": "t2.small"
             }
+        },
+        "LambdaBuilds": {
+            "Uploader": {
+                "CODE": "media-service/CODE/media-atom-uploader/media-atom-uploader.zip",
```

Comment:
minor: we could do an inline `Fn::Join` and avoid a mapping as its not providing much benefit here